mysql-test/suite/galera/t/pxc_implicit_commit.test

```diff
@@ -0,0 +1,27 @@
+#
+# Test the implicit commit of empty transaction (PXC-2963)
+#
+
+--source include/galera_cluster.inc
+--source include/wait_wsrep_ready.inc
+
+
+--connection node_2
+call mtr.add_suppression(".*The storage engine for the table doesn't support ENCRYPTION.*");
+
+--connection node_1
+call mtr.add_suppression(".*The storage engine for the table doesn't support ENCRYPTION.*");
+
+SET @@session.autocommit=0;
+CREATE TABLE t1(id INT, PRIMARY KEY(id));
+INSERT INTO t1 VALUES (1);
+COMMIT;
+--disable_warnings
```

Comment:
Is this section needed in the test?  disable warnings, commented out line, and then enable warnings?